### DISCUSSION TITLE
user invitation and forgot password email not working

### DISCUSSION THREAD

**Discussion by devops-cybersapient** [hystax/optscale#427] [Q&A]
I am using aws SES FOR SMTP . but when clicking forgot password and user invitation  i am not getting any emails. is it smtp issue or any other issue?

**Comment by devops-cybersapient** [comment]
server: email-smtp.us-east-1.amazonaws.com
  email: devops@test.io
  port: 587
  password: BKDAS1swv331GUusulug7wewrvefeUT+q2HpAiaz/mctIs7                        like this i have configured the smtp. i tried giving username also  .that is also not working

**Reply by SaurabhGupta168** [reply]
We also facing similar issues, the day we configured the tool SMTP was working but now it's not..

**Comment by stanfra** [comment]
Hi @devops-cybersapient, @SaurabhGupta168, issue solved in https://github.com/hystax/optscale/issues/428
